xml/install_caasp_heat_templates.xml

```diff
@@ -445,248 +452,240 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
-         </para>
-        </step>
-        <step>
-         <para>
-          Install certificates and find the <filename>.pem</filename> file.
-         </para>
-         <screen>&prompt.ardana;sudo zypper in ca-certificates-suse
-&prompt.ardana;rpm -ql ca-certificates-suse</screen>
-         <para>
-          The path to the cert is <filename>/usr/share/pki/trust/anchors/SUSE_Trust_Root.crt.pem</filename>
-         </para>
-        </step>
-       </substeps>
-      </step>
-      <step>
-       <para>
-        In <literal>system wide certificate</literal> during setup in Velum,
-        upload this certificate so you can bootstrap with CPI.
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>cpi_password</term>
+    <listitem>
+     <para>
+      Password of a valid user that has been set in &o_ident;.
      </para>
-      </step>
-     </substeps>
-    </step>
-    <step>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>cpi_monitor_max_retries</term>
+    <listitem>
      <para>
-      After the nodes come up, continue with the installation instructions in
-      <xref linkend="sec.heat.templates.install"/> following the admin node
-      cluster bootstrapping steps.
+      &o_netw; load balancer monitoring max retries. Default: 3
      </para>
-    </step>
-   </procedure>
-  </section>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>cpi_bs_version</term>
+    <listitem>
+     <para>
+      &o_blockstore; Block Storage API version. Possible values are v1, v2 , v3 or
+      auto. Default: auto
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>cpi_ignore_volume_az</term>
+    <listitem>
+     <para>
+      Ignore &o_blockstore; and &o_comp; availability zones. Default: true
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>dns_nameserver</term>
+    <listitem>
+     <para>
+      Set this to the IP address of a DNS nameserver accessible by the &caasp;
+      cluster.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+  <para>
+   Immediately after the &caasp; cluster comes online, and before bootstrapping,
+   install the latest &caasp; 3.0 Maintenance Update using the following steps.
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Register the &caasp; nodes for Maintenance Updates by following the
+     instructions in <xref linkend="sec.heat.templates.register"/>.
+    </para>
+   </step>
+   <step>
+    <para>
+     On each of the &caasp; nodes, install the latest Maintenance Update:
+    </para>
+    <screen>&prompt.sudo;transactional-update</screen>
+    <para>
+     Verify that the Velum image packages were updated:
+    </para>
+    <screen>&prompt.sudo;zypper se --detail velum-image
+i | sles12-velum-image | package    | 3.1.7-3.27.3  | x86_64 | update_caasp</screen>
+    <para>
+     Finally, reboot the node:
```

Comment:
```suggestion
Reboot the node:
```

Reply:
This is because you've got "Finally" on line 580. Remove one or the other, depending which is correct :)